### ISSUE TITLE
Add ATAC-Seq data support

### ISSUE THREAD

**Issue by heylf** [nf-core/scrnaseq#129]
### Description of feature

This goes hand in hand with the issue for CITE-Seq (https://github.com/nf-core/scrnaseq/issues/99). Does it make sense to also try to integrate ATAC-Seq data support in the pipeline as well? scATAC-Seq is also supported by Cell-Ranger's 10x. I will test out the pipeline on some real data anyway in a few days.

Yet, since the name is scrnaseq, it might be a bit hard for people to notice that the pipeline can also be used for scATAC-seq data.

**Comment by grst** [comment]
I just learned this would require the [`cellranger-arc`](https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/what-is-cell-ranger-arc) pipeline which seems independent of the regular cellranger command? 

So we would probably want to build a separate `cellranger-arc` module and support this as a separate aligner ("cellranger-arc" among "cellranger"/"kallisto"/"alevin-fry")? Any other thoughts about how to best implement this?

**Comment by heylf** [comment]
I think it's the best way to go for multiome. For scATAC alone, you could use cellranger itself. I wanted to implement the cellranger-arc module and am currently working on it.

**Comment by klkeys** [comment]
> For scATAC alone, you could use cellranger itself.
is this still true? it seems like you need `cellranger-atac`, whereas `cellranger-arc` handles the multiome data

scATAC data support should work once https://github.com/nf-core/modules/pull/2450 is merged

**Comment by grst** [comment]
closing in favor of https://github.com/nf-core/scrnaseq/issues/174
